cpp-unisocket/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM fedora:34
+
+ARG VERSION=4.2
+
+RUN dnf install -y gcc-c++ tar make wget bzip2 unzip cmake which boost-devel
+
+# install hazelcast cpp client
+RUN wget --quiet https://github.com/hazelcast/hazelcast-cpp-client/archive/v${VERSION}.zip && unzip -q v${VERSION}.zip && rm v${VERSION}.zip && cd hazelcast-cpp-client-${VERSION} && cmake -S . -B build && cmake --build build --verbose && cmake --install build && cd .. && rm -rf hazelcast-cpp-client-${VERSION}
+
+# build client application
+COPY . .
+RUN cmake -S . -B build -DCMAKE_BUILD_TYPE=Release && cmake --build build --verbose && cp build/cpp-unisocket .
+
+ENV LD_LIBRARY_PATH /usr/local/lib
```

Comment:
@ihsandemir was this necessary to be able to run the resulting executable?

Reply:
Well, Rafal put it for the kubernetes example, i believe he had problem during the runs. See [kubernetes example code](https://github.com/hazelcast-guides/kubernetes/blob/master/cpp/Dockerfile#L14). Normally, it should not be needed.

Reply:
I've tried and it worked without this line. I assume maybe it was something left over from the previous versions. 
@leszko, please report if you still have to do path manipulations like this to make the client work.